### PR TITLE
Update OV-XAI

### PR DESCRIPTION
**Pull request by negvet**
Align OV-XAI notebook with 1.1.0 release

### CODE REVIEW

File: .ci/spellcheck/.pyspelling.wordlist.txt [hunk 1]
```diff
@@ -948,6 +948,13 @@ wuerstchen
 WuerstchenDiffNeXt
 Würstchen
 XAI
+AISE
+AISEClassification
+AISEDetection
+WHITEBOX
+BLACKBOX
+Netron
+KDE
```
Comment (GalyaZalesskaya) [inline .ci/spellcheck/.pyspelling.wordlist.txt]: Shouldn't these names be organized alphabetically for the simplicity of adding new words?

File: notebooks/explainable-ai-2-deep-dive/README.md [hunk 2]
```diff
@@ -25,22 +25,25 @@ Example: Saliency map for `flat-coated retriever` class for MobileNetV3 classifi
 
 The tutorial consists of the following steps:
 
-- Run explainer in Auto-mode
+- Run explainer in `Auto` mode
```
Comment (GalyaZalesskaya) [inline notebooks/explainable-ai-2-deep-dive/README.md]: In L11-L12 here we talk only about IR models:
```
[OpenVINO™ Explainable AI (XAI)](https://github.com/openvinotoolkit/openvino_xai/) provides a suite of XAI algorithms for visual explanation of
[OpenVINO™](https://github.com/openvinotoolkit/openvino) Intermediate Representation (IR) models.
```
But now thanks to Songki, we have an option to insert_xai branch to PyTorch as well , even thought not to create `Explainer` using it. 
What do you think, should we mention PyTorch usecase in readmes?

**Review by GalyaZalesskaya — COMMENTED** [review]
Thank you for the great update :)

**Comment by negvet** [comment]
<p>not only list, tuple or numpy.array also would work here. That is why I use general term - container</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2394/discussion/#comment-2356444445'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by negvet** [comment]
<p>Right, good point, will update</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2394/discussion/#comment-2356444390'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by negvet** [comment]
<p>Sure, will fix</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2394/discussion/#comment-2356444461'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by negvet** [comment]
<p>Will fix</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2394/discussion/#comment-2356444499'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by negvet** [comment]
<p>I like this idea. Adopted</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2394/discussion/#comment-2356444534'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by negvet** [comment]
<p>there is no link as for now</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2394/discussion/#comment-2356444620'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by negvet** [comment]
<p>Exactly, will use explanation.save()</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2394/discussion/#comment-2356444685'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: .ci/spellcheck/.pyspelling.wordlist.txt (hunk 1 above)
Comment (negvet) [reply .ci/spellcheck/.pyspelling.wordlist.txt]: Right, updated

File: notebooks/explainable-ai-2-deep-dive/README.md (hunk 2 above)
Comment (negvet) [reply notebooks/explainable-ai-2-deep-dive/README.md]: I mentioned `PyTorch` in the `insert_xai` section

**Comment by GalyaZalesskaya** [comment]
<p>Are we talking about the same RISE paper? <a href="https://arxiv.org/abs/1806.07421" target="_blank">[1806.07421] RISE: Randomized Input Sampling for Explanation of Black-box Models (arxiv.org)</a></p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2394/discussion/#comment-2356444620'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: notebooks/explainable-ai-2-deep-dive/README.md [hunk 3]
```diff
@@ -25,22 +25,25 @@ Example: Saliency map for `flat-coated retriever` class for MobileNetV3 classifi
...
 - Specify preprocess and postprocess functions
-- Run explainer in White-box mode
+- Run explainer in `WHITEBOX` mode
   - Insert XAI branch to use updated model in own pipelines
```
Comment (GalyaZalesskaya) [inline notebooks/explainable-ai-2-deep-dive/README.md]:
```suggestion
  - Insert XAI branch to IR or Pytorch model to use updated model in own pipelines
```

**Comment by negvet** [comment]
<p>Ah, sure, I misunderstood</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2394/discussion/#comment-2356444620'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by negvet** [comment]
<p>Added</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2394/discussion/#comment-2364256282'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by GalyaZalesskaya — APPROVED** [review]

**Review by goodsong81 — APPROVED** [review]
LGTM

**Comment by negvet** [comment]
@eaidova could you please take a look?

**Comment by negvet** [comment]
@eaidova could you please take a look?

to support `--no-deps`, I separately installed scipy (it is imported and used by some methods)

**Review by eaidova — APPROVED** [review]

**Pull request merged**